### PR TITLE
Add documentation for performance testing with Docker

### PR DESCRIPTION
**Pull request by awoods**
Resolves: https://jira.lyrasis.org/browse/FCREPO-3550

To test, follow directions in docs/Performance-Testing.md .

### CODE REVIEW

File: docs/Performance-Testing.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- default username / password: fedoraAdmin / fedoraAdmin
+Grafana will be available at: http://host:3000
+- default username / password: admin / admin
+
+# System Setup
+
+1. [Install Docker](https://docs.docker.com/get-docker/)
+1. Create a Fedora home directory for all repository persistence. This configuration expects:
+   ```
+   mkdir /data/fcrepo-home
```
Comment (pwinckles) [inline docs/Performance-Testing.md]: `mkdir -p /data/fcrepo-home` maybe?

File: docs/fcrepo.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+fcrepo.activemq.directory=/data/activemq
+fcrepo.home=/data
```
Comment (pwinckles) [inline docs/fcrepo.properties]: Shouldn't this be `/data/fcrepo-home`? I'm a little confused as to what's being mapped where.

Comment (awoods) [reply docs/fcrepo.properties]: In this setup, `/data` in the Fedora Docker container is Fedora home: https://github.com/fcrepo-exts/fcrepo-docker/pull/13/files#diff-dec5aef1eb56327aab4a81fa2f9a69ecbede5b3ea1413e93ccb7f1dd7ea75018R17

File: docs/Performance-Testing.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- default username / password: fedoraAdmin / fedoraAdmin
+Grafana will be available at: http://host:3000
+- default username / password: admin / admin
+
+# System Setup
+
+1. [Install Docker](https://docs.docker.com/get-docker/)
+1. Create a Fedora home directory for all repository persistence. This configuration expects:
+   ```
+   sudo mkdir -p /opt/fcrepo-home
```
Comment (dwilcox) [inline docs/Performance-Testing.md:22]: I'm not sure if this applies to other operating systems, but on Mac OS I had to add /opt to the list of directories that can be mounted into Docker by opening the Docker application Preferences and navigating to Resources -> File Sharing. After adding /opt I hit Apply & Restart.

**Review by dwilcox — APPROVED** [review]
I tested the PR and everything now works as expected.

**Review by pwinckles — APPROVED** [review]

**Pull request merged**